lib/src/components/NxTextInput/NxTextInput.scss

```diff
@@ -6,17 +6,18 @@
  */
 @import '../../scss-shared/nx-variables';
 @import '../../scss-shared/_nx-form-validation.scss';
+@import '../../scss-shared/_nx-container-helpers.scss';
```

Comment:
Is this import still needed here?

Reply:
> Is this import still needed here?

No, and neither was `nx-variables`: https://github.com/sonatype/sonatype-react-shared-components/pull/57/commits/08e04017637c8308f996adc15728b6b7f0016ad1